### ISSUE TITLE
make install fails on FreeBSD

### ISSUE THREAD

**Issue by Gandalf-** [minio/mc#2787]
## Expected behavior
`gmake install` is able to copy result binary to `$GOPATH/bin`

## Actual behavior
`gmake install` on FreeBSD fails because `cp` is given a non POSIX flag `-u`

## Steps to reproduce the behavior
```
I leaf@alder ~/w/g/s/g/m/mc (master=)> gmake install
Checking dependencies
Building MinIO binary to './mc'
Installing mc binary to '/home/leaf/working/go/bin/mc'
cp: illegal option -- u
usage: cp [-R [-H | -L | -P]] [-f | -i | -n] [-alpsvx] source_file target_file
       cp [-R [-H | -L | -P]] [-f | -i | -n] [-alpsvx] source_file ... target_directory
gmake: *** [Makefile:69: install] Error 64
```

## mc version
- Not really applicable, using latest commit
```
I leaf@alder /m/g/c/g/s/g/m/mc (master=)> git log | head
commit eb9493458ea73d9a066767b1e03b9e44725bab71
Author: Anis Elleuch <vadmeste@users.noreply.github.com>
Date:   Thu May 30 01:55:15 2019 +0100

    autocomplete: update github.com/posener/complete to include macOS fix (#2784)

    https://github.com/posener/complete/commit/afda8e00c6d48e7d20bff281c321bc8aca00699a
    is sent to fix a rare macOS issue with installing completion command.

    The commit will require a specific commit in posener/complete to have our fix
```

## System information
```
I leaf@alder ~/w/g/s/g/m/mc (master=)> uname -a
FreeBSD alder 11.2-STABLE FreeBSD 11.2-STABLE #0 r325575+9a3c7d8b53f(HEAD): Wed Mar 27 12:41:58 EDT 2019     root@mp20.tn.ixsystems.com:/freenas-releng/freenas/_BE/objs/freenas-releng/freenas/_BE/os/sys/FreeNAS.amd64  amd64
```

**Comment by Gandalf-** [comment]
This is also a problem for the main `minio` project too. 

```
I leaf@alder /m/g/c/g/s/g/m/minio (master=)> gmake install
Checking dependencies
Building minio binary to './minio'
go: finding github.com/hashicorp/hcl/hcl/ast latest
go: finding github.com/hashicorp/hcl/hcl latest
Installing minio binary to '/home/leaf/working/go/bin/minio'
cp: illegal option -- u
usage: cp [-R [-H | -L | -P]] [-f | -i | -n] [-alpsvx] source_file target_file
       cp [-R [-H | -L | -P]] [-f | -i | -n] [-alpsvx] source_file ... target_directory
gmake: *** [Makefile:77: install] Error 64
```

**Comment by harshavardhana** [comment]
This is already fixed in MinIO project we don't use cp -u anymore, same needs to be done for mc

**Comment by harshavardhana** [comment]
Should be fixed by #2788

**Comment by kannappanr** [comment]
Closing this issue as fixed by #2788
